dist/spec.json

```diff
@@ -22081,6 +22077,38 @@
         "Brand"
       ]
     },
+    "ThemeSettings": {
+      "properties": {
+        "emailTemplateTouchPointVariant": {
+          "$ref": "#/definitions/EmailTemplateTouchPointVariant"
+        },
+        "endUserDashboardTouchPointVariant": {
+          "$ref": "#/definitions/EndUserDashboardTouchPointVariant"
+        },
+        "errorPageTouchPointVariant": {
+          "$ref": "#/definitions/ErrorPageTouchPointVariant"
+        },
+        "primaryColorContrastHex": {
+          "type": "string"
+        },
+        "primaryColorHex": {
+          "type": "string"
+        },
+        "secondaryColorContrastHex": {
+          "type": "string"
+        },
+        "secondaryColorHex": {
+          "type": "string"
+        },
+        "signInPageTouchPointVariant": {
+          "$ref": "#/definitions/SignInPageTouchPointVariant"
+        }
+      },
+      "x-okta-parent": "#/definitions/Theme",
```

Comment:
any value in extending `Theme`?

Reply:
I am working on this PR right now to change this up a bit.  There wont be a parent anymore